client/components/Signup.js

```diff
@@ -55,14 +53,13 @@ const SignupForm = (props) => {
   );
 };
 
-/**
- * CONTAINER
+// CONTAINER
+/*
```

Comment:
don't think we should use both a single-line comment and multi-line comment in one spot

Reply:
Agreed. Didn't know if we wanted to keep the other notes. I'll remove it:)